ui/src/simpledesk.cpp

```diff
@@ -405,6 +405,28 @@ uchar SimpleDesk::getAbsoluteChannelValue(uint address)
 void SimpleDesk::setAbsoluteChannelValue(uint address, uchar value)
 {
     m_engine->setValue(address, value);
+
+    if (m_viewModeButton->isChecked() == true)
+    {
+        quint32 fxID = m_doc->fixtureForAddress(address);
+        if (fxID == Fixture::invalidId())
+            return;
+
+        Fixture* fixture = m_doc->fixture(fxID);
+        if (fixture == NULL)
+            return;
```

Comment:
This can be replaced by
`Fixture *fxi = m_doc->fixture(m_doc->fixtureForAddress(address));`